javascript/javascript1/week1/answers.js

```diff
@@ -0,0 +1,83 @@
+/* 
+//************************************************ 
+STEP2 answer
+//************************************************ 
+
+Alejandro Gispert
+https://www.freecodecamp.org/AlejandroGispert
+
+*/
+
+//************************************************ 
+//Age-ify (A future age calculator) 
+//************************************************ 
+
+const yearOfBirth = 1986
+const yearFuture = 2045
+
+const age = yearFuture - yearOfBirth
+
+console.log("You will be " + age +" years old in " + yearFuture)
+
+//************************************************ 
+//Goodboy-Oldboy (A dog age calculator)
+//************************************************ 
+
+
+const dogYearOfBirth = 2010
+const dogYearFuture = 2045
+let dogYear = dogYearFuture - dogYearOfBirth
+const shouldShowResultInDogYears = true
+let years = "human years"
+
+if (shouldShowResultInDogYears) {
+years = "dog years"
+   dogYear *= 5
+}
+console.log("Your dog will be " + dogYear +" "+ years +" old in " + dogYearFuture )
+
+
+
+//************************************************ 
+//Housey pricey (A house price estimator)
+//************************************************ 
+const length = 5   
+const width = 11
+const depth = 8
+
+const Peter = [8,10,10]
+const Julia =[length,width,depth]
```

Comment:
I like that you put the values into an array to avoid lot's of typing. Yet, the variable names `Peter` and `Julia` should be `camelCase` meaning their first character should be lower case. Then further, `peter` and `julia` are not that much better names, either. Maybe `peterDimensions` and `juliaDimensions` are better?

Reply:
done